openwisp-monitoring/files/monitoring.agent

```diff
@@ -195,6 +197,11 @@ send_data() {
 			done
 			# retry sending same data again in next cycle
 			[ "$failures" -eq "$MAX_RETRIES" ] && break
+            # pause for a while after every 10 successful request sent
```

Comment:
watch out from indentation mismatches like this

Reply:
Sorry MB, formatting issue will fix it